example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -368,15 +368,17 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
       if (beforeAllError) fail(beforeAllError);
       if (subTestSetupError) fail(subTestSetupError);
     });
-
-    it('publishes the granule metadata to CMR', async () => {
+    it('pretends to work', () => {
```

Comment:
I'm assuming this was proofing/diagnostic?     Looks like this should be removed.

Reply:
this is intentional, it's here to trigger the PostToCMrTask beforeAll() function